src/projector.ts

```diff
@@ -180,18 +180,25 @@ export const createProjector: ProjectorFactory = compose<ProjectorMixin, Project
 		render(this: Projector): VNode {
 			const projectorData = projectorDataMap.get(this);
 			const childVNodes: VNode[] = [];
-			this.children.forEach((child) => childVNodes.push(child.render()));
+			this.children.forEach((child) => {
+				// Workaround for https://github.com/facebook/immutable-js/pull/919
+				if (child) {
+					childVNodes.push(child.render());
+				}
+			});
 			const props = this.getNodeAttributes();
 			props.afterCreate = projectorData.afterInitialCreate;
 			return h(projectorData.tagName, props, childVNodes);
 		},
-		attach(this: Projector, { type, tagName = 'div' }: AttachOptions = {}): Promise<Handle> {
+		attach(this: Projector, { type, tagName }: AttachOptions = {}): Promise<Handle> {
 			const projectorData = projectorDataMap.get(this);
 			if (projectorData.state === ProjectorState.Attached) {
-				return projectorData.attachPromise;
+				return projectorData.attachPromise || Promise.resolve(noopHandle);
 			}
 			projectorData.boundRender = this.render.bind(this);
-			projectorData.tagName = tagName;
+			if (tagName !== undefined) {
+				projectorData.tagName = tagName;
```

Comment:
There are no tests that pass options to `attach()`, so previously when we assigned the default while destructuring the argument this line would get covered by the existing test. This is now uncovered like a bunch of other stuff in the function that relies on options being passed.